src_kt/project/WORKSPACE

```diff
@@ -1 +0,0 @@
-../../WORKSPACE
```

Comment:
@jasonwyatt may still need/want this, deleted intentionally?

Reply:
This is intended to go in as part of #4220, it causes breakage on Windows.